Resources/config/plugins.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="httplug.plugin.content_length" class="Http\Client\Plugin\ContentLengthPlugin" public="false" />
+        <service id="httplug.plugin.decoder" class="Http\Client\Plugin\DecoderPlugin" public="false">
+            <argument />
```

Comment:
We do not have tests that cover this. You are correct that the bundle lacks tests.